packages/server/src/store/index.ts

```diff
@@ -141,4 +383,9 @@ Project.init(
 Project.belongsToMany(Dataset, { through: "DatasetProject" });
 Dataset.belongsToMany(Project, { through: "DatasetProject" });
 
+Project.belongsToMany(Checkpoint, { through: "ProjectCheckpoint" });
+Project.belongsToMany(Export, { through: "ProjectExport" });
+Project.belongsToMany(Video, { through: "ProjectVideo" });
+Project.belongsToMany(Test, { through: "ProjectTest" });
```

Comment:
Is the relationship that a Checkpoint has an export instead of a project has an export?

Same with test. Export has a Test and Test has a video.